lib/install/validate/validator.go

```diff
@@ -404,6 +414,9 @@ func (v *Validator) network(ctx context.Context, input *data.Data, conf *metadat
 			Nameservers: dns,
 			Pools:       pools,
 		}
+		if input.BridgeNetworkName == net {
+			v.NoteIssue(errors.Errorf("bridge network %s is used for container network %s, please choose different network", input.BridgeNetworkName, name))
```

Comment:
the bridge network must not be shared with another network role - %s also mapped as container network %s